src/index.html

```diff
@@ -8,6 +8,12 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="container">
+      <div class="star--1"></div>
```

Comment:
at the moment you have five containers, each will have colored stars and the number of colored stars should be equal to the number on the class modifier. the first row of stars should have 0 stars colored. so before `<div class="star--1"></div>` you need to add another div. something like <div class="stars"></div>